packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1530,6 +1536,12 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   }
 
   void _handleSelectionChanged(TextSelection selection, RenderEditable renderObject, SelectionChangedCause cause) {
+    // We return early if the selection is not valid. This can happen when the
+    // text of [EditableText] is updated at the same time as the selection is
```

Comment:
Use /// for highlighting

Reply:
This is a [comment](https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences) though and not a doc comment. So I'll keep it as it is.